src/views/Completed.vue

```diff
@@ -332,6 +331,15 @@ export default defineComponent({
             handler: async () => {
               let orderList = JSON.parse(JSON.stringify(this.completedOrders.list))
 
+              // if there are orders with tracking required and label image present
```

Comment:
Here we only need to exclude those orders for which tracking is required but the shipping label is missing. Please correct the logic.